utils/utils.go

```diff
@@ -687,3 +741,32 @@ func SendError(w http.ResponseWriter, statusCode int, message string) {
 	w.WriteHeader(statusCode)
 	json.NewEncoder(w).Encode(map[string]string{"error": message})
 }
+
+func updateUploadStatus(db *sql.DB, videoID string, status types.UploadStatus) error {
+	log.Printf("Updating upload status to -1 for video_id %s", videoID)
```

Comment:
This will always log -1 even if the function is used to update a different status.